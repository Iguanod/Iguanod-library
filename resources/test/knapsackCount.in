333
433.509 936.481 827.045 965.053 843.757 385.802 436.448 542.955 699.041 471.333 412.498 77.960 740.761 109.404 813.961 976.493 590.432 476.350 222.809 786.640 261.868 418.899 617.112 479.471 832.440 580.233 912.778 173.088 117.202 185.250 979.158 852.279 72.351 108.619 119.332 217.088 443.803 506.265 61.972 92.469 279.619 423.797 471.788 969.378 834.639 587.240 248.923 374.373 365.974 420.120 463.136 576.770 141.913 1030.630 1006.242 923.920 912.433 220.087 1046.091 331.013 355.708 975.369 134.281 377.062 386.931 555.362 896.963 779.460 363.341 907.691 174.296 945.501 632.526 595.183 864.585 417.832 484.977 414.572 93.663 800.629 137.385 858.925 326.845 228.632 838.349 634.161 453.657 701.870 804.017 449.802 982.570 110.014 726.673 418.807 789.204 62.836 276.439 636.929 144.793 589.150 845.067 620.535 484.231 428.981 517.258 651.254 147.676 952.403 367.085 542.115 702.532 806.680 351.852 330.089 336.592 491.852 915.171 739.050 494.273 669.901 490.127 779.222 81.159 167.973 147.139 821.124 531.543 725.910 759.545 977.858 617.219 906.528 547.987 403.361 636.594 1015.659 356.380 733.877 269.057 674.949 578.652 921.510 782.660 879.820 553.420 69.139 320.363 418.796 758.516 765.420 390.506 199.603 846.553 773.829 668.709 295.059 896.761 501.922 971.092 605.810 429.968 538.089 814.570 278.372 891.893 400.068 243.007 550.789 84.907 814.404 174.142 964.442 1037.028 906.860 793.565 541.214 277.187 415.504 909.493 986.229 482.763 601.820 487.700 278.878 325.940 457.755 524.811 171.694 908.719 445.010 79.060 287.901 285.294 843.282 868.758 478.155 193.088 61.377 978.953 579.062 826.931 454.310 493.217 813.559 663.763 588.534 656.020 890.144 954.745 868.470 178.942 738.211 419.881 967.327 967.800 1046.503 374.049 793.043 520.875 233.770 540.879 549.823 822.562 775.257 694.045 640.395 555.168 189.568 1004.360 836.563 719.073 132.794 592.504 514.084 247.153 557.526 405.390 854.202 398.166 661.506 672.873 878.375 349.809 393.256 796.030 618.904 390.281 472.940 361.256 860.160 655.470 851.118 711.201 780.093 928.357 355.784 370.992 786.512 494.809 676.297 572.200 515.429 758.721 466.359 332.767 308.365 974.361 687.203 112.464 674.135 650.876 86.559 502.477 949.065 429.111 600.735 518.406 121.008 1023.326 181.397 931.309 980.764 983.902 944.217 710.406 213.764 249.836 1031.591 949.726 1046.913 1009.706 823.059 863.283 718.548 240.763 145.872 976.326 516.330 134.117 390.432 140.847 734.745 426.370 592.920 986.787 157.588 495.792 806.525 229.342 468.443 937.250 462.131 398.336 222.710 709.047 59.165 386.881 260.165 392.264 637.616 256.474 351.402 763.545 422.870 371.188 305.734 869.631 649.126 771.891 954.328 989.435 861.622 990.243 
383 886 777 915 793 335 386 492 649 421 362 27 690 59 763 926 540 426 172 736 211 368 567 429 782 530 862 123 67 135 929 802 22 58 69 167 393 456 11 42 229 373 421 919 784 537 198 324 315 370 413 526 91 980 956 873 862 170 996 281 305 925 84 327 336 505 846 729 313 857 124 895 582 545 814 367 434 364 43 750 87 808 276 178 788 584 403 651 754 399 932 60 676 368 739 12 226 586 94 539 795 570 434 378 467 601 97 902 317 492 652 756 301 280 286 441 865 689 444 619 440 729 31 117 97 771 481 675 709 927 567 856 497 353 586 965 306 683 219 624 528 871 732 829 503 19 270 368 708 715 340 149 796 723 618 245 846 451 921 555 379 488 764 228 841 350 193 500 34 764 124 914 987 856 743 491 227 365 859 936 432 551 437 228 275 407 474 121 858 395 29 237 235 793 818 428 143 11 928 529 776 404 443 763 613 538 606 840 904 818 128 688 369 917 917 996 324 743 470 183 490 499 772 725 644 590 505 139 954 786 669 82 542 464 197 507 355 804 348 611 622 828 299 343 746 568 340 422 311 810 605 801 661 730 878 305 320 736 444 626 522 465 708 416 282 258 924 637 62 624 600 36 452 899 379 550 468 71 973 131 881 930 933 894 660 163 199 981 899 996 959 773 813 668 190 95 926 466 84 340 90 684 376 542 936 107 445 756 179 418 887 412 348 172 659 9 336 210 342 587 206 301 713 372 321 255 819 599 721 904 939 811 940 
12846
