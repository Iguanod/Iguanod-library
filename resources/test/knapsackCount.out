16990.634
000000000001010000101000000111001111001110000010100010000100101010000010000000101011000000010001101000010010000010100000001110000000000000100000011000010001000000010010101000000000000110010011100011000000000000100000000100000001000100100000001000000000000110000000110010010000010100000110000000110010100001001000010111001000010000000
